vowpalwabbit/core/src/reductions/automl.cc

```diff
@@ -545,6 +573,15 @@ void interaction_config_manager::revert_config(example* ec) { ec->interactions =
 template <typename CMType>
 void automl<CMType>::offset_learn(multi_learner& base, multi_ex& ec, CB::cb_class& logged, uint64_t labelled_action)
 {
+  VW::reductions::automl::interaction_vec_t* incoming_interactions = ec[0]->interactions;
+  for (VW::example* ex : ec)
+  {
+    _UNUSED(ex);
```

Comment:
removable?

Reply:
without that line, the compiler complaints on Release mode because the assert gets compiled out and we have warnings as errors, so we get an error of unused var.